### DISCUSSION TITLE
[Feature suggestion] Slot number in info.toml

### DISCUSSION THREAD

**Discussion by Segtendo** [Raytwo/ARCropolis#296] [Ideas]
### Have you read the disclaimer before proceeding?

- [X] I have read the disclaimer.

### What would the feature do?

I think it would be a helpful ability to add a new field to the info.toml file: a field to mention which slot the currently-selected mod goes over.

### How should it be implemented?

Add a field to the info.toml field to specify information for a slot.

For example, say the config file is like this:
`display_name = "Byleth (F): Yukari Takeba"`
`authors = "MadMax1960"`
`version = "1.0"`
`slot = "c03"`
`description = """`
`"""`
`category = "Fighter"`

Then, the result would be this:
![slot config idea](https://user-images.githubusercontent.com/76792055/163087722-be56289c-c0c3-4910-b987-845c53bc5a8c.jpg)

The slot could be an optional field, as stuff like effects and music wouldn't affect costume slots.

**Comment by Raytwo** [comment]
I'm honestly not sure about this, could you perhaps list some of the benefits of this so we can consider it?

**Reply by Segtendo** [reply]
It could help a user tell what slot a mod is over. To be honest, I don't know how else to describe it, as I didn't think it would warrant much response.

Been thinking about it today, and I guess I could just not be lazy and add the slot to the description instead. I think I'll just ask to have it closed it for now, as I can't come up with any other reasonings.

**Reply by Raytwo** [reply]
Sounds good, thanks
